content/en/blog/2022-09-05-august-update/index.md

```diff
@@ -0,0 +1,351 @@
+took us almost 4 months, from the first RFC version to shipping OCI
+support today. And a special thanks to Rashed and the whole VMware Tanzu
+team for the excellent collaboration!
+
+{{< tweet user=stefanprodan id=1557754198648913921 >}}
+
+### Security news
+
+We are continuously putting effort into the security story of Flux. One
+cornerstone of this is fuzzing of all code. As
+[promised](/blog/2022/02/security-more-confidence-through-fuzzing/#whats-next),
+we started [transitioning our fuzz
+tests](https://github.com/fluxcd/flux2/issues/2417) to the
+native Go implementation.
+
+We are happy to say that we managed to contribute back to Google's
+`oss-fuzz` improving Go Native Fuzz implementation as well during this
+effort ([patch
+1](https://github.com/google/oss-fuzz/pull/8238), [patch
+2](https://github.com/google/oss-fuzz/pull/8285)).
+
+### Flagger 1.22.2
+
+[Flagger 1.22.2](https://github.com/fluxcd/flagger/releases/tag/v1.22.2)
+received a patch release as well during August. It fixes a bug related
+to scaling up the canary deployment when a reference to an auto-scaler
+is specified.
+
+Furthermore, it contains updates to packages used by the project,
+including updates to Helm and grpc-health-probe used in the load-tester.
+
+A number of CVEs originating from its dependencies were fixed as well.
+
+### Flux Ecosystem
+
+#### Flux Subsystem for Argo
+
+Flux added `OCIRepository` as a new kind of Source in its recent release.
+The new version of Flux Subsystem for Argo (FSA) brings these good bits
+of Flux to Argo CD. The team has also recently upgraded FSA to Argo CD
+v2.2.12 to contain recent security bug fixes again. This version of Flux
+Subsystem for Argo requires Flux v0.32.0 to install.
+
+{{< imgproc fsa Resize 600x >}}
+{{< /imgproc >}}
+
+#### Terraform-controller
+
+The team has released [TF-controller
+v0.11](https://github.com/weaveworks/tf-controller/blob/main/CHANGELOG.md#v0110)
+which now supports Flux OCIRepository. To use Flux `OCIRepository` with
+TF-controller, you're required to upgrade Flux to v0.32+.
+
+{{< imgproc tf-controller1 Resize 600x >}}
+{{< /imgproc >}}
+
+In addition to the new `OCIRepository` support, the TF-controller team is
+glad to announce that the performance of TF-controller has been improved
+significantly. Now the controller is greatly scalable to reconcile and
+provision high volumes of Terraform modules concurrently. The team has
+recently tested the controller with 1,500 Terraform modules.
+
+{{< imgproc tf-controller2 Resize 600x >}}
+{{< /imgproc >}}
+
+#### Weave GitOps
+
+[https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.2](https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.2)
+
+[https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.3](https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.3)
+
+[https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.4](https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.4)
+
```

Comment:
ping @ozamosi @chanwit @JamWils

Reply:
Ok. I'll just add a small note that a number of releases came out and that people should upgrade and consult the changelog.